package/utils/util-linux/Makefile

```diff
@@ -8,12 +8,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=util-linux
-PKG_VERSION:=2.37.3
+PKG_VERSION:=2.38
```

Comment:
I would prefer if you split this into to commits, one doing the upgrade and an other doing the move to meson.

Reply:
Done.